_posts/2015-03-03-Polls-API-Blueprint-Tutorial-Examples.md

```diff
@@ -0,0 +1,87 @@
+---
+title: New API Blueprint tutorial
+excerpt: Learning API Blueprint has never been easier.
+layout: post
+date: 2015-03-25 16:00:00 +0000
+author: zdenek
+published: true
+comments: true
+---
+
+Everything at Apiary starts with API Blueprint and API Blueprint starts with its tutorial. Since providing superb and concise documentation is our priority, there is no other document that should stand more for our values.
```

Comment:
..'Providing superb and concise documentation is our priority and API Blueprint should be a direct representation of these goals.'